test/manifest.ts

```diff
@@ -1085,7 +1085,7 @@ describe('Manifest', () => {
           python: {
             'release-type': 'python',
             'package-name': 'foolib',
-            'release-draft': true,
+            draft: true,
```

Comment:
I believe this will technically be breaking right? If any folks happened to have found the `release-draft` config already?

Reply:
you are correct. when I first wrote/submitted this it was very shortly after the factory/cli wireup so I felt ok at that time that no one was using it yet. but now that it's been a minute and this is setup in maybe a few places it could break people who are using this knob... I could just revert it and keep `release-draft`?